flytekit/clis/flyte_cli/main.py

```diff
@@ -1818,6 +1818,7 @@ def _extract_and_register(
 
     flyte_entities_list = _extract_files(project, domain, version, file_paths, patches)
     for id, flyte_entity in flyte_entities_list:
+        _click.secho(f"Registering {id}", fg="green")
```

Comment:
```suggestion
        _click.secho(f"Registering {id}", fg="yellow")
```

Reply:
green makes you feel that it was successful.
Ideally I would like this

Registering {name}..... [OK] | [FAILED]